test/browser/step_definitions/CICCheckAnswersEdit.step.js

```diff
@@ -2,21 +2,69 @@ const { Given, When, Then} = require("@cucumber/cucumber");
 
 const { expect } = require("chai");
 
-const { CheckDetailsPage, DateOfBirthPageEdit, DateOfBirthPage }  = require("../pages");
+const { CheckDetailsPage, DateOfBirthPageEdit, DateOfBirthPage, NameEntryPageEdit }  = require("../pages");
 
+//Merged DOB edit and NE edit
 Given(/^the user has navigated to Check my Answers page$/, async function () {
     const cdPage = new CheckDetailsPage(await this.page);
 
     expect(await cdPage.isCurrentPage()).to.be.true;
 
 });
 
+When(/^the name entry edit link is clicked$/, async function () {
+  const cdPage = new CheckDetailsPage(await this.page);
+
+  await cdPage.changeName();
+
+});
+
+Then(/^the user is navigated back to the name entry page$/, async function () {
+  const nameEntryPage = new NameEntryPageEdit(await this.page);
+
+  expect(await nameEntryPage.isCurrentPage()).to.be.true;
+
+  //await nameEntryPage.clearFields();
```

Comment:
Can you please remove commented out code?

Reply:
sure, will do now.